README.md

```diff
@@ -132,6 +132,19 @@ type of the parameter:
   the content will be URL-encoded (see [form posts](#form-posts) below)
 * For all other types, the object will be serialized as JSON.
 
+#### Bufferering and the `Content-Length` header
```

Comment:
I should fix this typo. "Bufferering" :joy:

Reply:
Adds charm 😉